src/metal.cs

```diff
@@ -304,6 +348,21 @@ public partial interface MTLComputeCommandEncoder : MTLCommandEncoder {
 		[Abstract]
 		[Export ("setBytes:length:atIndex:")]
 		void SetBytes (IntPtr bytes, nuint length, nuint index);
+
+		[iOS (10,0), TV (10,0), NoWatch, Mac (10,12)]
+		[Abstract]
+		[Export ("setStageInRegion:")]
+		void SetStageInRegion (MTLRegion region);
+
+		[iOS (10,0), TV (10,0), NoWatch, NoMac]
+		[Abstract]
+		[Export ("updateFence:")]
+		void UpdateFence (IMTLFence fence);
```

Comment:
yeah, I was not sure enough of the API but if it means Update *the* fence (and not Update *using* the fence) then the original name was better.

@VincentDondain @dalexsoto thoughts ?